src/dfmt/main.d

```diff
@@ -265,11 +265,15 @@ else
                 if (!config.isValid())
                     return 1;
                 File f = File(path);
-                buffer = new ubyte[](cast(size_t) f.size);
-                f.rawRead(buffer);
-                if (inplace)
-                    output = File(path, "wb");
-                format(path, buffer, output.lockingTextWriter(), &config);
+                // ignore empty files
+                if (f.size)
```

Comment:
Didn't we already merge these changes?

Reply:
yes we did but into the wrong branch :-/

Reply:
ah I see :)